homeassistant/components/conversation/chat_log.py

```diff
@@ -183,6 +183,20 @@ class ChatLog:
     llm_api: llm.APIInstance | None = None
     delta_listener: Callable[[ChatLog, dict], None] | None = None
 
+    @property
+    def continue_conversation(self) -> bool:
+        """Return if the conversation should continue."""
+        if not self.content:
+            return False
+
+        last_msg = self.content[-1]
+
+        return (
+            last_msg.role == "assistant"
+            and last_msg.content is not None  # type: ignore[union-attr]
+            and last_msg.content.strip()[-1] == "?"  # type: ignore[union-attr]
```

Comment:
If you extract it to a local variable maybe you can use the  ignore type once?

Reply:
The problem is that `last_msg.role == "assistant"` is not picked up by MyPy to mean that it can only be an AssistantContent object. So any interaction needs to be guarded.